create_test.go

```diff
@@ -1059,3 +1059,60 @@ func TestCreateCreateContainer(t *testing.T) {
 		assert.NoError(err)
 	}
 }
+
+func TestCopyParentCPUSetFail(t *testing.T) {
+	assert := assert.New(t)
+
+	cgroupsPath, err := ioutil.TempDir(testDir, "cgroups-path-")
+	if err != nil {
+		t.Fatalf("Could not create temporary cgroups directory: %s", err)
+	}
+	defer os.RemoveAll(cgroupsPath)
+
+	err = copyParentCPUSet(cgroupsPath, testDir)
+	assert.Error(err)
+}
+
+func TestCopyParentCPUSetSuccessful(t *testing.T) {
+	if os.Geteuid() != 0 {
+		// Create cgroup directory must be root.
+		t.Skip(testDisabledNeedRoot)
+	}
+
+	assert := assert.New(t)
+
+	cgroupsRootPath := findCgroupRoot()
+	if cgroupsRootPath == "" {
+		t.Skip(testDisabledNeedCgroup)
+	}
+
+	cgroupsCPUSetRoot := filepath.Join(cgroupsRootPath, "cpu")
+	if _, err := os.Stat(cgroupsCPUSetRoot); err != nil && os.IsNotExist(err) {
+		t.Skip(testDisabledNeedCgroup)
+	}
+
+	cpusetCpusPath := filepath.Join(cgroupsCPUSetRoot, "cpuset.cpus")
+	if _, err := os.Stat(cpusetCpusPath); err != nil && os.IsNotExist(err) {
+		t.Skip(testDisabledNeedCgroup)
+	}
+
+	cpusetMemsPath := filepath.Join(cgroupsCPUSetRoot, "cpuset.mems")
+	if _, err := os.Stat(cpusetMemsPath); err != nil && os.IsNotExist(err) {
+		t.Skip(testDisabledNeedCgroup)
+	}
+
+	cgroupsPath, err := ioutil.TempDir(cgroupsCPUSetRoot, "cgroups-path-")
+	if err != nil {
+		t.Fatalf("Could not create temporary cgroups directory: %s", err)
+	}
+	defer os.RemoveAll(cgroupsPath)
```

Comment:
That's good, it won't fail on distros where cpuset is not created under the cpu cgroup.
However, since we're running our CI on Ubuntu and FC26, it will prevent us from testing the positive path of `copyParentCPUSet()` on our CI (The above tests will always fail and thus fall into skip). Could you please add another test where we run exactly the same code path, except we do it under `/sys/fs/cgroup/cpuset`, so that we unit test both cases ? This does not have to be part of this PR (provided the code coverage is fine), it can be in a next one after we merge that one (I want to merge it asap to get you guys rolling).